### ISSUE TITLE
Invisible background fillings are required?

### ISSUE THREAD

**Issue by sator-imaging** [google/material-design-icons#1833]
Hi there,
I realize that material icons SVG has invisible background that is same size to View Box. I'm not sure is there any reason to draw them?

--

I've tried to remove it by programmatically for my use but it's so hard. there are many patterns...

```svg
# most simple, easily removed by code --> av\volume_down_alt\materialicons\24px.svg
<rect fill="none" height="24" width="24"/>

# some rect has 'x' and/or 'y' attribute --> av\volume_down_alt\materialicons\20px.svg
<rect fill="none" height="20" width="20" x="0"/>

# draw rect by path, simple --> av/videocam/materialicons/24px.svg
<path d="M0 0h24v24H0V0z" fill="none"/>

# some icon draws path from other than top-left corner, and what's this magic number 0.87?
# --> navigation\unfold_less\materialiconstwotone\24px.svg
<path d="M24 0v24H0V0h24z" fill="none" opacity=".87"/>

# drawing rect twice? --> hardware\keyboard\materialiconssharp\24px.svg
<path d="M0 0h24v24H0V0zm0 0h24v24H0V0z" fill="none"/>

# and more patterns, some icon using style instead of fill attribute
<rect style="fill:none" .../>
```

**Comment by omkarkulkarni2704** [comment]
Exactly! These extra elements help ensure icons display consistently across different screen resolutions and use cases. Tools like SVGO can streamline this process by removing redundant parts, or you can use XML parsing libraries to programmatically clean up SVGs by targeting specific elements like <rect> and <path> with fill="none".

**Comment by omkarkulkarni2704** [comment]
Exactly! These extra elements help ensure icons display consistently across different screen resolutions and use cases. Tools like SVGO can streamline this process by removing redundant parts, or you can use XML parsing libraries to programmatically clean up SVGs by targeting specific elements like and with fill="none".

**Comment by omkarkulkarni2704** [comment]
@sator-imaging Exactly! These extra elements help ensure icons display consistently across different screen resolutions and use cases. Tools like SVGO can streamline this process by removing redundant parts, or you can use XML parsing libraries to programmatically clean up SVGs by targeting specific elements like and with fill="none".
